core/include/kernel/dt.h

```diff
@@ -33,13 +34,18 @@
  * @reset: Device reset identifier (positive value) or DT_INFO_INVALID_CLOCK
  * @interrupt: Device interrupt identifier (positive value) or
  * DT_INFO_INVALID_INTERRUPT
+ * @irq_type: IRQ_TYPE_* value parsed from interrupts properties or
```

Comment:
prefer `type` over `irq_type`. `irq` can be confusing on Arm (optee native (secure) interrupts can be IRQs or FIQs. See foreign/native interrutp).

Reply:
Ok, renamed it and changed irq -> interrupt to be more generic.